src/js/dqa.jsx

```diff
@@ -0,0 +1,222 @@
+var React = require('react'),
+    _ = require('underscore');
+
+var FixedDataTable = require('fixed-data-table');
+var Table = FixedDataTable.Table;
+var Column = FixedDataTable.Column;
+
+var isColumnResizing;
+var width = 70;
+var columnWidths = {'code': width, 'desc': 3*width, 'status': 2*width, 'num': width};
+
+var SortTypes = {
+  ASC: 'ASC',
+  DESC: 'DESC',
+};
+        
+var DQA = React.createClass({
+    propTypes: {
+        content: React.PropTypes.string,
+        width: React.PropTypes.number,
+        data: React.PropTypes.object,
+    },
+
+    getDefaultProps: function() {
+        return {
+            content: '',
+            width: 500,
+            data: {},
+        };
+    },
+
+    getInitialState: function() {
+        return {
+            isCollapsed: false,
+            rows: this._getRows(),
+            sortBy: null,
+            sortDir: null,
+        };
+    },
+
+    _getRows: function() {
+        var rows = [];
+        var codes = Object.keys(this.props.data);
+        for(var i=0; i<codes.length; i++) {
+            var stats = Object.keys(this.props.data[codes[i]]);
+            for(var j=0; j<stats.length; j++) {
```

Comment:
That is really only necessary if you don't have control over the object and it has static properties or methods. To me using a keys array and then indexing into it [looks confusing](https://github.com/PEDSnet/pedsnet-catalogue/pull/1/files#diff-aaf94063b9204adb1c65d16ecad52fa7R47).